### DISCUSSION TITLE
Pregunta 2: ¿Basureros limitados ó ilimitados?

### DISCUSSION THREAD

**Discussion by try-except** [IIC2133-PUC/2024-1#182] [Tarea 3] [answered]
Hola, tengo una duda respecto a la disponibilidad de los basureros de esta pregunta. En el segundo ejemplo del enunciado dice "Se puede usar 6 veces cualquier tipo de basural que aporta 2", pero en el input hay solo 3 basureros de capacidad 2, entonces no entiendo por qué el input trae una cantidad determinada de basureros tipo 2 si la solución considera basureros ilimitados. En la cápsula también se menciona que los basureros son limitados, lo que me genera aún más confusión.

**Comment by 4n1t4** [comment, marked as answer]
hola, en el contexto de la tarea mejor consideralos como uso de basureros ilimitados. De esta forma los inputs de ejemplo estarían correctos.
